src/components/playback/skipsegment.ts

```diff
@@ -78,6 +83,11 @@ class SkipSegment extends PlaybackSubscriber {
 
             void elem.offsetWidth;
 
+            const osdVisible = document.activeElement && focusManager.isCurrentlyFocusable(document.activeElement);
+            if (options.focus && !osdVisible) {
```

Comment:
A more appropriate name, imo. 
```suggestion
            const hasFocus = document.activeElement && focusManager.isCurrentlyFocusable(document.activeElement);
            if (options.focus && !hasFocus) {
```